recipe/meta.yaml

```diff
@@ -14,21 +14,25 @@ source:
     folder: ext/robin_map
 
 build:
-  number: 0
+  number: 1
 
 outputs:
   - name: nanobind-abi
     version: {{ abi_version }}
+    requirements:
+      run_constrained:
+        # conflict with nanobind releases before the nanobind-abi package was added
+        - nanobind >=2.4
```

Comment:
This doesn't need updating, it's to prevent `nanobind==2.1.0, nanobind-abi==16` from being a valid solution.

Reply:
An alternative is a repodata patch on old nanobind builds to add constraint on `nanobind-abi <0`.